src/Attribute/TargetAware.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace Zalas\PHPUnit\Globals\Attribute;
+
+trait TargetAware
+{
+    protected int $target = 0;
```

Comment:
I guess this could be private. Any reason to make it protected?

Reply:
Now private :)